appservice/src/tree/AppSettingTreeItem.ts

```diff
@@ -32,7 +32,7 @@ export class AppSettingTreeItem extends AzExtTreeItem {
         super(parent);
         this._client = client;
         this._key = key;
-        this._value = value;
+        this._value = value.trim();
```

Comment:
The 3 issues linked to this PR show that the whitespace is trimmed on the platform level.  There's no mention of it in the API documentation though.  https://docs.microsoft.com/en-us/rest/api/appservice/webapps/updateapplicationsettings.

The portal behaves the same way.  You can add a setting with leading and trailing whitespace, but if you refresh the page, it'll be removed from the setting.

Reply:
If there's no official docs of the behavior, then it sounds like a bug in the platform rather than an intentional design choice IMO. In that case, I would rather close those 3 issues as "won't fix" rather than perpetuate the bug ourselves. That way, if/when the platform team fixes their bug - we will automatically get the fix.

For example, sometimes app settings are passwords and I could see some random person ending their password with a space.

Reply:
That's fine with me and makes sense.